ktor-io/common/src/io/ktor/utils/io/bits/MemoryFactory.kt

```diff
@@ -12,6 +12,11 @@ import kotlin.contracts.contract
  * By default, if neither [offset] nor [length] specified, the whole array is used.
  * An instance of [Memory] provided into the [block] should be never captured and used outside of lambda.
  */
+/**
+ * TODO: Solve design problems
+ * 1. length has no default
+ * 2. no inline -> can't suspend inside block
```

Comment:
This wasn't inline just because of some compiler bug. What a shame it's still not fixed

Reply:
added to comment